src/main/java/com/yahoo/memory/NioBits.java

```diff
@@ -122,38 +122,32 @@ static boolean isPageAligned() {
   }
 
   //RESERVE & UNRESERVE BITS MEMORY TRACKING COUNTERS
-  static void reserveMemory(final long capacity) {
+  static void reserveMemory(final long allocationSize) {
```

Comment:
If you look at `java.nio.Bits.tryReserveMemory(size, cap)` line 703+ you will note the comment:

>         // -XX:MaxDirectMemorySize limits the total capacity rather than the
>         // actual memory usage, which will differ when buffers are page
>         // aligned.

I was following the comment recommendation by submitting the `capacity` to NIO_BITS_RESERVE_MEMORY_METHOD not the `allocationSize`.    Why did you feel the need to change this?